src/alliteratedActors.test.js

```diff
@@ -35,6 +35,14 @@ test('Ten people', () => {
     .toEqual('Vnnchyxwx Vxot');
 });
 
+test('Uppercase letter in the middle of the word', () => {
+  expect(alliteratedActors(['Wn Tjuyol', 'Eyd Gjvxav', 'Wjuybt Hc Fiwyr',
```

Comment:
1. can't find uppercase letter in the middle of the word in this test case. 
2. let's rephrase each description to explain what scenario is being tested. Maybe half of them are not really needed and they are testing essentially same scenarios.

Reply:
done

Reply:
shorten input array to minimal possible input to test scenario 'Should work only with first letter uppercase'.

Reply:
done